docs/coordinates/index.rst

```diff
@@ -318,13 +319,21 @@ use this option if you do not need sub-arcsecond accuracy for your coordinates::
 ..
   EXAMPLE END
 
+.. testsetup::
+
+    >>> from astropy.coordinates import EarthLocation, SkyCoord
+    >>> loc = EarthLocation.of_site('Royal Observatory Greenwich', force_builtin=True)
```

Comment:
I like this idea of a test setup, but couldn't for the locations at least it be done simpler with:
```
from astropy.coordinates import EarthLocation
EarthLocation._get_site_registry(force_builtin=True)
```

Reply:
Re: caching -- I don't think #9162 affects `of_site`?

Reply:
`EarthLocation._get_site_registry(force_builtin=True)` is problematic because I need to use `force_download` to undo it, which defeats the purpose of not hitting data server.

Reply:
I think in the other thread you're converging on hard-coding, which would be OK, though how about even more cheekily setting `EarthLocation._site_registry = None` at the end of tests? Probably overkill...

p.s. Yes, caching is only for `SkyCoord`. But only useful if the sites works too. My sense: go with @bsipocz suggestion and have an issue about perhaps prepopulating the cache instead.

Reply:
I don't know how to pre-populate cache on CI. I was under the impression that we need to store the tarball of the cache somewhere and then have the CI pull it but I have no idea how to set up the tarball in the first place.

Yes, I am in the works to hardcode stuff. Will push a commit soon.